lib/presentation/widget/shared_space/shared_space_widget.dart

```diff
@@ -276,7 +276,7 @@ class _SharedSpaceWidgetState extends State<SharedSpaceWidget> {
     } else {
       return ListView.builder(
         key: Key('shared_spaces_list'),
-        padding: ResponsiveWidget.getPaddingForScreen(context),
+        padding: ResponsiveWidget.getPaddingListItemForScreen(context),
```

Comment:
To be honest, I am not a fan of static functions. Why not create a class `ResponsiveUtil` with a method like that and inject it here?

Reply:
oke. boy

Reply:
oke